feeds/utils.R

```diff
@@ -110,6 +115,20 @@ get_org_topics <- function(org) {
     )
 }
 
+get_usr_topics <- function(usr) {
+  # User should be lower case
+  usr <- tolower(usr)
+  get_list_repos(usr) %>%
+    dplyr::filter(
+      !private,
+      carpentries_usr == usr
```

Comment:
@tkphd I get the feeling here that the filter here should remain `carpentries_org` (which is useless in this case)

Reply:
Fixed in https://github.com/hpc-carpentry/hpc-carpentry.github.io/commit/87cda90ead2d4dbee83c5f80927b050344d5f8cd